### PR TITLE
Fix links with no thumbnail having previous link's thumbnail

### LINKED ISSUE #1971: Image not correctly updated when hovering over two adjacent links
**Describe the bug**
When you have two links right adjacent to each other (with no gap), e.g. on separate lines right below each other, and then you hover over these links with the link tooltips enabled, should one link have an image and the other one no image, then you will notice the image is not cleared when going from the link with the image to the link without the image.

**To reproduce**
See GIF above.

**Screenshots**

fg2Z5CV

**Chatterino version**
Chatterino Nightly 2.2.2 (commit c7a2f4cc9, )

**Operating system**
Windows 10

**Additional information**

### PR DESCRIPTION
**Pull request by qooq69**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
Fixes  #1971

### CODE REVIEW

**Review by Felanbird — APPROVED** [review]
Functionality works as intended. 👍
before:
image

after:
image

**Review by zneix — APPROVED** [review]
works

File: src/providers/LinkResolver.cpp [hunk 1]
```diff
@@ -26,32 +26,34 @@ void LinkResolver::getLinkInfo(
...
+            QString response;
+            QString linkString = url;
+            ImagePtr thumbnail = nullptr;
+            if (statusCode == 200)
+            {
+                response = root.value("tooltip").toString();
+
+                if (root.contains("thumbnail"))
                     thumbnail =
                         Image::fromUrl({root.value("thumbnail").toString()});
```
Comment (pajlada) [inline src/providers/LinkResolver.cpp:42]:
```suggestion
                if (root.contains("thumbnail"))
                {
                    thumbnail =
                        Image::fromUrl({root.value("thumbnail").toString()});
                }
```

**Review by pajlada — CHANGES_REQUESTED** [review]
Other than this nitpick, looks good

**Review by pajlada — APPROVED** [review]

**Pull request merged**